.github/workflows/create_issues_for_muted_tests.yml

```
@@ -55,4 +55,25 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ env.GH_TOKEN }}
         run: |
-            python .github/scripts/create_or_update_pr.py append_pr_body --pr_number=${{ github.event.pull_request.number || github.event.inputs.pr_number  }} --body=issues_info.txt
\ No newline at end of file
+            python .github/scripts/create_or_update_pr.py append_pr_body --pr_number=${{ github.event.pull_request.number || github.event.inputs.pr_number  }} --body=${{env.GITHUB_WORKSPACE}}/issues_info.txt
+      
+      - name: Comment PR
+        uses: actions/github-script@v7
+        
+        with:
+          github-token: ${{ env.GH_TOKEN }}
+          script: |
+            const fs = require('fs');
+            const path = require('path');
+            
+            const workflowUrl = `https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}`;
+            const filePath = path.join(process.env.GITHUB_WORKSPACE, 'issues_info.txt');
+            const bodyText = fs.readFileSync(filePath, 'utf8');
+            const completeBody = `Collected in workflow [#${{ github.run_number }}](${workflowUrl})\n\n${bodyText}`;
+      
+            github.rest.issues.createComment({
+              issue_number: ${{ github.event.pull_request.number || github.event.inputs.pr_number  }},
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: completeBody
+            });
```

Давай пустую строчку сделаем в конце
```suggestion
            });

```

---

done